components/ContactForm.js

```diff
@@ -0,0 +1,15 @@
+var ContactForm = React.createClass({
+	propTypes: {
+		contact: React.PropTypes.object.isRequired,
+	},
+	render: function() {
+		return (
+			<form className={'ContactForm'}>
+				<input type={'text'} placeholder={'First Name'} value={this.props.contact.firstName}/>
```

Comment:
możesz użyć atrybutów normalnie (bez {}) dla type="text" i placeholder="First Name".
Dodanie wyrażenia wewnątrz {...} uruchamia kod JS, w przypadku type i placeholder nie ma potrzeby - to są statyczne wartości tekstowe.